src/Xrefcheck/CLI.hs

```diff
@@ -69,6 +73,17 @@ addTraversalOptions TraversalConfig{..} (TraversalOptions ignored) =
   , ..
   }
 
+newtype VerifyOptions = VerifyOptions
+  { voCheckLocalhost :: Maybe Bool
+  }
+
+addVerifyOptions :: VerifyConfig -> VerifyOptions -> VerifyConfig
+addVerifyOptions VerifyConfig{..} (VerifyOptions checkLocalhost) =
+  VerifyConfig
+  { vcCheckLocalhost = fromMaybe vcCheckLocalhost checkLocalhost
+  , ..
+  }
```

Comment:
Why `RecordWildCards`, thought? You can update it with old haskell98:

```haskell
addVerifyOptions :: VerifyConfig -> VerifyOptions -> VerifyConfig
addVerifyOptions verifyConfig (VerifyOptions checkLocalhost) =
  verifyConfig
    { vcCheckLocalhost = fromMaybe vcCheckLocalhost checkLocalhost
    }
```

Reply:
Just in same style as here
https://github.com/serokell/xrefcheck/blob/60eccc0386ce61a81690677fe5cdae88e071cff3/src/Xrefcheck/CLI.hs#L70